### ISSUE TITLE
Remove Gulp as a dependency

### ISSUE THREAD

**Issue by ericwbailey** [a11yproject/a11yproject.com#989]
I love Gulp, but I love less external dependencies more. I initially used it for the redesign to get things up and running quickly.

The tasks we're using Gulp for are relatively straightforward, namely asset minification, concatenation, post-processing, and Browsersync. Because of this, there is an opportunity to do a lot of this via node scripts. Big-picture, I'd also like to lean more on Eleventy's capabilities to do the same—we're already using it for things like `addPassthroughCopy`.

To remove Gulp, we'd need to:

- [ ] Create node scripts for each Gulp task, and ensure feature parity.
- [ ] See what tasks can be used in Eleventy directly.
- [ ] Update `npm run a11yproject` to make sure it works.
- [ ] Update Netlify to reflect these changes.
- [ ] Remove references to Gulp in our documentation.
- [ ] Do a little happy dance.

**Comment by ericwbailey** [comment]
@esjay You had mentioned wanting to tackle this previously. How does this sound to you?

**Comment by esjay** [comment]
> @esjay You had mentioned wanting to tackle this previously. How does this sound to you?

@ericwbailey put me in coach!

**Comment by ericwbailey** [comment]
Woo!

**Comment by esjay** [comment]
Due to a dependency upon how eleventy processes incremental builds, we can get about 95% of the way towards this, but the build output will suffer due to eleventy's need to [rebuild the entire site](https://github.com/11ty/eleventy/issues/108) whenever a passthrough file copy operation occurs. Currently, this is on [eleventy's Incremental enhancement board/roadmap](https://github.com/orgs/11ty/projects/3). 

After discussing with @ericwbailey, we decided to pause existing work on this PR until something moves on the parallelized/incremental build front with eleventy.

**Comment by SaptakS** [comment]
The updates to eleventy's [incremental usage](https://www.11ty.dev/docs/usage/incremental/) in v2.x might make this possible now?

**Comment by ericwbailey** [comment]
Yes! Upgrading this site to use Eleventy 2 is on my list, and this should address that, exactly like what you said, @SaptakS.

@esjay: Is this something you'd be interested in pairing on?
